autoloader.php

```diff
@@ -11,21 +11,72 @@
 	$vendorDirectory = __DIR__ . '/../..';
 }
 
-if (is_file($vendorAutoloader = $vendorDirectory . '/hoa/core/Core.php')) {
-	require_once $vendorDirectory . '/hoa/core/Core.php';
+$hoa = array(
```

Comment:
This is fragile. Imagine we are adding a new dependency. This mapping will break. Consider that if something is null present, it acts as `null`.